tests/consensus_tests/utils.py

```diff
@@ -122,28 +122,36 @@ def start_peer(peer_dir: Path, log_file: str, bootstrap_uri: str, port=None, ext
     http_port = get_port() if port is None else port + 2
     _occupy_port(http_port)
 
-    env = {
-        **get_env(p2p_port, grpc_port, http_port),
-        **extra_env
-    }
     test_log_folder = init_pytest_log_folder()
     log_file = open(f"{test_log_folder}/{log_file}", "w")
+    this_peer_consensus_uri = get_uri(p2p_port)
     print(f"Starting follower peer with bootstrap uri {bootstrap_uri},"
           f" http: http://localhost:{http_port}/cluster, p2p: {p2p_port}")
 
-    this_peer_consensus_uri = get_uri(p2p_port)
-    if reinit:
-        proc = Popen([get_qdrant_exec(), "--bootstrap", bootstrap_uri, "--uri", this_peer_consensus_uri, "--reinit"], env=env,
-                     cwd=peer_dir, stdout=log_file)
+    args = [get_qdrant_exec()]
+    env = {
+        **get_env(p2p_port, grpc_port, http_port),
+        **extra_env
+    }
+
+    if uris_in_env:
+        if "QDRANT_BOOTSTRAP" not in env:
+            env["QDRANT_BOOTSTRAP"] = bootstrap_uri
+        if "QDRANT_URI" not in env:
+            env["QDRANT_URI"] = this_peer_consensus_uri
```

Comment:
I'd insert variables into `env` unconditionally.

```suggestion
        env["QDRANT_BOOTSTRAP"] = bootstrap_uri
        env["QDRANT_URI"] = this_peer_consensus_uri
```

Reply:
Makes it a bit less likely to unintentionally break/alter tests, IMO.

Reply:
Okay!

I did it this way in case we ever pass different values from a test. But that may be a case of premature optimization.

I'll remove it. If we ever need it we can add it back.